.eslintignore

```diff
@@ -1,3 +1,4 @@
 **/*.json
 **/node_modules/*
-**/dist/*
\ No newline at end of file
+**/dist/*
+/lib/js/app.js
```

Comment:
There's an alternative way to have the linter ignore rules for certain files that I'd recommend instead, because we use this other method for other files (see files in rpc folder)
I suggest placing a comment at the top of app.js which gets picked up by the eslinter to ignore the file.